packages/viz/src/BaseChart/BaseChart.tsx

```diff
@@ -1,15 +1,15 @@
 import { forwardRef, useEffect, useRef, useState } from "react";
 import { useForkRef } from "@mui/material/utils";
 import ReactECharts from "echarts-for-react/lib/core";
-import { AriaComponent } from "echarts/components";
+import { AriaComponent, TitleComponent } from "echarts/components";
 import * as echarts from "echarts/core";
 import { CanvasRenderer } from "echarts/renderers";
 
 import { useVizTheme } from "../hooks";
 import { HvChartCommonProps, HvEChartsOption } from "../types/common";
 
 // Register chart components
-echarts.use([CanvasRenderer, AriaComponent]);
+echarts.use([CanvasRenderer, AriaComponent, TitleComponent]);
```

Comment:
Can we not?
We're adding "bloat" to the base chart, to add custom text in a docs sample that doesn't add much to it.
Besides, it doesn't adjust in dark mode, and feels misaligned 

![image](https://github.com/user-attachments/assets/c4d2a7f7-d947-4869-99ce-b9a14b37901c)